packages/ETLoad.package/ETLObjectHeader.class/instance/classOop.st

```diff
@@ -1,3 +1,5 @@
 accessing
 classOop
+	"Oop to the class of the object. Nil if the class is compacted."
```

Comment:
OOP is a abbreviation for "object oriented pointer", and is basically the same as a pointer or reference in other languages. The term OOP seems to be quite commonly used in the smalltalk world, which is why we chose to use it here. See the section "Basic OOP Concepts and Terminology" on [this page](http://web.cecs.pdx.edu/~harry/musings/SmalltalkOverview.html) for more info.